store/plasmaStore.go

```diff
@@ -6,34 +6,59 @@ import (
 	"github.com/FourthState/plasma-mvp-sidechain/utils"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/ethereum/go-ethereum/rlp"
+	"io"
 	"math/big"
 )
 
 type PlasmaStore struct {
-	KVStore
+	kvStore
+}
+
+type Block struct {
+	plasma.Block
+	TMBlockHeight uint64
+}
+
+type block struct {
```

Comment:
this is because plasma.Block implements the RLP interface. Serializing `Block` in this case will ignore `TMBlockHeight`. However as a named parameter, it applies it recursively